### PR TITLE
Feat 2.0.x/abc#71765 select no available choices option

### PR DESCRIPTION
**Pull request by NathanHGit**
# Description

Setting default text to "No available choices" when there is no option for a select. 

## Useful links

Ticket: [71765 - ABC - Add "no available choices" text when no content is available in a select](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/71765)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I tried to load an empty select and checked if the display was correct. I also checked that the text is set again if values are added and then removed.

## Screenshots

peek3

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AntoineRelief** [comment]
@NathanHGit 
I was not clear enough
it's not in the input I'd like the text, but in the popup

kind of what we would see there:
https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/templates/#toc-no-data-template

**Review by AntoineRelief — CHANGES_REQUESTED** [review]
need to put the text in the popup, not in the input directly
as we would see there:
https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/templates/#toc-no-data-template

**Review by matheus-relief — APPROVED** [review]
Just added an icon and tweaked style a bit, looks better to me, but feel free to revert the changes! Other than that, don't see any issue!

image

File: libs/ui/src/lib/select-menu/select-menu.component.ts [hunk 1]
```diff
@@ -90,6 +90,7 @@ export class SelectMenuComponent
    * @param renderer Renderer2
    * @param viewContainerRef ViewContainerRef
    * @param overlay Overlay
+   * @param translate Angular translate service
```
Comment (AntoineRelief) [inline libs/ui/src/lib/select-menu/select-menu.component.ts]: @NathanHGit 
cc @matheus-relief 
please check linting
this one does not pass the test, as the constructor does not need the translate service

**Pull request merged**